code/go/0chain.net/smartcontract/dbs/event/users.go

```diff
@@ -0,0 +1,87 @@
+package event
+
+import (
+	"errors"
+	"fmt"
+
+	"0chain.net/smartcontract/dbs"
+	"gorm.io/gorm"
+)
+
+type Users struct {
+	gorm.Model
+	TransactionID uint   `json:"transaction_id"`
+	Balance       int64  `json:"balance" msgpack:"b"`
+	UserID        string `json:"user_id" gorm:"uniqueIndex"`
+}
+
+func (edb *EventDb) GetUsers(usrID string) (*Users, error) {
```

Comment:
We are fetching a singular user, not an array of users. So the appropriate method name would be GetUser().